// Error Testing...
// app.get('/', (req, res) => {
//   //   throw new Error('Hello Error......');
//   //   throw new ApiError(400, 'ApiError Error......');
// });

// // async error
// app.get('/', async (req, res, next) => {
//   Promise.reject(new Error('Unhandled Promise Rejection'));
// });

// // sync error
// app.get('/', (req, res, next) => {
//   // console.log(x);
//   throw new Error('Testing error logger...');
// });
